spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -87,7 +87,10 @@ def team_id=(team_id)
 
       response = request(:post) do |req|
         req.url "ra/v1/session/webSession"
-        req.body = { contentProviderId: team_id }.to_json
+        req.body = {
+          contentProviderId: team_id,
+          dsId: Time.now.to_i.to_s # https://github.com/fastlane/fastlane/issues/6711
```

Comment:
@milch my thoughts exactly. may work now, but not in future. i think it's worth doing right if it doesn't slow things down too much, which I don't think it does. @KrauseFx merged in #6715 which uses the dsId from user details